File: Assignment/app/src/main/java/com/example/assignment/MainActivity2.java
package com.example.assignment;

import androidx.appcompat.app.AppCompatActivity;

import android.widget.EditText;
import android.os.Bundle;
import android.widget.TextView;
import android.content.Intent;


public class MainActivity2 extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);

        Intent intent=getIntent();
        String message =intent.getStringExtra(MainActivity.EXTRA_MESSAGE);

        TextView textView=findViewById(R.id.textView);
        textView.setText(message);
    }


}

File: HelloWorldKotlin/app/src/main/java/com/example/helloworldkotlin/MainActivity2.kt
package com.example.helloworldkotlin

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import java.awt.font.TextAttribute

class MainActivity2 : AppCompatActivity() {
     lateinit var plus : Button
     lateinit var minus : Button
     lateinit var text : TextView
     var num =0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)
        plus = findViewById(R.id.btnPlus)
        minus = findViewById(R.id.btnMinus)
        text = findViewById(R.id.textView)
        plus.setOnClickListener{add()}
        minus.setOnClickListener{minus()}

    }
    fun add() {
        if(num > 0){
            num ++
            text.text=num.toString()
            text.setTextColor(Color.GREEN)
        }else{
            num ++
            text.text=num.toString()
            text.setTextColor(Color.RED)
        }
    }
    fun minus() {
        if(num > 0){
            num --
            text.text=num.toString()
            text.setTextColor(Color.GREEN)
        }else{
            num --
            text.text=num.toString()
            text.setTextColor(Color.RED)
        }

    }

}